chart/tests/test_basic_helm_chart.py

```diff
@@ -36,13 +36,19 @@ def test_basic_deployments(self):
                     'metadata': 'AA',
                 },
                 'labels': {"TEST-LABEL": "TEST-VALUE"},
+                "fullnameOverride": "TEST-BASIC",
```

Comment:
Why do we need to pass `fullnameOverride` ? What will be the value if we don't pass it?

Reply:
Sounds good, we can keep it as it is

Reply:
the change would look like the following if you want to remove the `fullnameOverride`:

```
--- a/chart/tests/test_basic_helm_chart.py
+++ b/chart/tests/test_basic_helm_chart.py
@@ -36,21 +36,20 @@ class TestBaseChartTest(unittest.TestCase):
                     'metadata': 'AA',
                 },
                 'labels': {"TEST-LABEL": "TEST-VALUE"},
-                "fullnameOverride": "TEST-BASIC",
             },
         )
         list_of_kind_names_tuples = [
             (k8s_object['kind'], k8s_object['metadata']['name']) for k8s_object in k8s_objects
         ]
         assert list_of_kind_names_tuples == [
-            ('ServiceAccount', 'TEST-BASIC-flower'),
-            ('ServiceAccount', 'TEST-BASIC-create-user-job'),
-            ('ServiceAccount', 'TEST-BASIC-migrate-database-job'),
-            ('ServiceAccount', 'TEST-BASIC-redis'),
-            ('ServiceAccount', 'TEST-BASIC-scheduler'),
-            ('ServiceAccount', 'TEST-BASIC-statsd'),
-            ('ServiceAccount', 'TEST-BASIC-webserver'),
-            ('ServiceAccount', 'TEST-BASIC-worker'),
+            ('ServiceAccount', 'TEST-BASIC-airflow-flower'),
+            ('ServiceAccount', 'TEST-BASIC-airflow-create-user-job'),
+            ('ServiceAccount', 'TEST-BASIC-airflow-migrate-database-job'),
+            ('ServiceAccount', 'TEST-BASIC-airflow-redis'),
+            ('ServiceAccount', 'TEST-BASIC-airflow-scheduler'),
+            ('ServiceAccount', 'TEST-BASIC-airflow-statsd'),
+            ('ServiceAccount', 'TEST-BASIC-airflow-webserver'),
+            ('ServiceAccount', 'TEST-BASIC-airflow-worker'),
             ('Secret', 'TEST-BASIC-postgresql'),
             ('Secret', 'TEST-BASIC-airflow-metadata'),
             ('Secret', 'TEST-BASIC-airflow-result-backend'),
```

Reply:
No, that's fine :) we can keep it as it is